### PR TITLE
Remove Launchdarkly

### PR DESCRIPTION
**Pull request by thetif**
Resolves the need for LaunchDarkly

### Description
Removes LaunchDarkly from the application. Turns all previously flagged code "on". Fully enables MMIS and adminCheck. Created a work around for the new code that shows or hides New Apdland. Created work around for sysadmin and just hardcoded access for the username 'sysadmin'.

### Chromatic Link
https://www.chromatic.com/pullrequest?appId=61d5b948cf6f17003a12bf77&number=4681

### Significant changes or possible side effects
Enabled all beta feature code.

### Steps to manually verify this change

1. Login as betauser and see that New Apdland is not available
2. Login as norole and see that New Apdland is available
3. Login as sysadmin and see that you have access to all states, click on one state
4. Verify that the Create new APD button isn't visible
5. Login as state admin, verify that you can create an MMIS APD and that it looks right

### This pull request is ready to code review when

- [x] Automated tests are updated (and all tests are passing)
- [ ] Chromatic link added above
- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor

### This pull request is ready to test when

- [ ] Code has been reviewed by someone other than the original author

### This pull request is ready to review when QA has

- [ ] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [ ] Verified all updated pages with the WAVE tool
- [ ] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/containers/ApdApplication.js [hunk 1]
```diff
@@ -94,7 +88,7 @@ const ApdApplication = ({
             <ApdPageRoutes apdId={paramApdId} />
           </div>
         </main>
-        {adminCheckFlag === true && <AdminCheckPanel />}
+        <AdminCheckPanel />
```
Comment (tbolt) [inline web/src/containers/ApdApplication.js:91]: 😄

**Review by tbolt — APPROVED** [review]
Looks good

**Review by amyd11 — APPROVED** [review]
Nice work taking all this out

**Review by mirano-darren — APPROVED** [review]
Looks good to me! All accounts behave as expected

**Pull request merged**